<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cleanroast</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="buttonConnection">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../content/resources.qrc">
          <normaloff>:/icons/disconnect.png</normaloff>
          <activeon>:/icons/connect.png</activeon>:/icons/disconnect.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonRoasting">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../content/resources.qrc">
          <normaloff>:/icons/record.png</normaloff>
          <activeon>:/icons/stop.png</activeon>:/icons/record.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="RoastGraph" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="buttonDevices">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../content/resources.qrc">
          <normaloff>:/icons/devices.png</normaloff>:/icons/devices.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonProfiles">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../content/resources.qrc">
          <normaloff>:/icons/graphs.png</normaloff>:/icons/graphs.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button1C">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../content/resources.qrc">
          <normaloff>:/icons/crack.png</normaloff>:/icons/crack.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button2C">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../content/resources.qrc">
          <normaloff>:/icons/2crack.png</normaloff>:/icons/2crack.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buittonComment">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../content/resources.qrc">
          <normaloff>:/icons/comment.png</normaloff>:/icons/comment.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="DeltaGraph" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RoastGraph</class>
   <extends>QWidget</extends>
   <header>cr.roastgraph.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DeltaGraph</class>
   <extends>QWidget</extends>
   <header>cr.deltagraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../content/resources.qrc"/>
 </resources>
 <connections/>
</ui>
